ios/Sources/TimetableFeature/TimetableDetailScreen.swift

```diff
@@ -0,0 +1,245 @@
+                        .frame(width: Const.footerIconSize, height: Const.footerIconSize)
+                }
+                .padding()
+                .background(AssetColor.primaryPale.color)
+                .cornerRadius(6)
+
+                HStack(spacing: 10) {
+                    AssetImage.iconPlayCircleFilled.image
+                        .foregroundColor(.white)
+                        .frame(width: 24, height: 24)
+
+                    Text(L10n.TimetableScreen.Detail.watchThisVideo)
+                        .font(.headline)
+                        .bold()
+                        .foregroundColor(.white)
+
+                    Spacer()
+
+                    AssetImage.iconChevron.image
+                        .renderingMode(.template)
+                        .foregroundColor(.white)
+                        .frame(width: Const.footerIconSize, height: Const.footerIconSize)
+                }
+                .padding()
+                .background(AssetColor.primary.color)
+                .cornerRadius(6)
+            }
+            .padding(8)
+            .background(AssetColor.Background.primary.color)
+        }
+    }
+}
+
+private enum DesciptionType: CaseIterable {
+    case schedule
+    case length
+    case category
+    case language
+    case difficulty
+
+    var image: SwiftUI.Image {
+        switch self {
+        case .schedule:
+            return AssetImage.iconSchedule.image
+        case .length:
+            return AssetImage.iconTime.image
+        case .category:
+            return AssetImage.iconCategory.image
+        case .language:
+            return AssetImage.iconLanguage.image
+        case .difficulty:
+            return AssetImage.iconDifficulty.image
+        }
+    }
+
+    func value(timetable: TimetableItem) -> String {
+        switch self {
+        case .schedule:
+            return "\(timetable.startsAt.formatToDate) \(timetable.startsAt.formatToTime) ~ \(timetable.endsAt.formatToTime)"
+        case .length:
+            return "\(getMinDiffFrom(startAt: timetable.startsAt, endAt: timetable.endsAt))min"
+        case .category:
+            return timetable.category
+        case .language:
+            return "日本語" // TODO: FIXME
+        case .difficulty:
+            return "(Difficulty of the session)" // TODO: FIXME
+        }
+    }
+
+    func getMinDiffFrom(startAt: Date, endAt: Date) -> Int {
+        let diff = Int(endAt.timeIntervalSince1970 - startAt.timeIntervalSince1970)
+        let hours = diff / 3600
+        let minutes = (diff - hours * 3600) / 60
+        return minutes
+    }
+}
+
+private extension Date {
+    var formatToDate: String {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "yyyy-MM-dd"
+        return formatter.string(from: self)
+    }
+
+    var formatToTime: String {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "HH:mm"
+        return formatter.string(from: self)
+    }
```

Comment:
If use this, please define these formatter with stored property. Because initialize formatter'a performance is not good.